sbt/src/sbt-test/dependency-management/snapshot-resolution/build.sbt

```diff
@@ -37,9 +37,9 @@ lazy val dependent = project.
     // Ignore the inter-project resolver, so we force to look remotely.
     resolvers += sharedResolver,
     fullResolvers := fullResolvers.value.filterNot(_==projectResolver.value),
-    libraryDependencies += "com.badexample" % "badexample" % "1.0-SNAPSHOT",
+    libraryDependencies += "com.badexample" % "badexample" % "1.0-SNAPSHOT"
     // Setting this to false fails the test
-    updateOptions := updateOptions.value.withLatestSnapshots(true)
+    // updateOptions := updateOptions.value.withLatestSnapshots(true)
```

Comment:
This option has gone back and forth, so I am leaving that in in case we're going to default it to `false` again.